_includes/head.html

```diff
@@ -0,0 +1,9 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | relative_url }}">
+  {%- feed_meta -%}
```

Comment:
What is this for - an RSS feed?

Reply:
Yes. The content of head.html is replacing the default head.html from minima, e.g. https://github.com/jekyll/minima/blob/master/_includes/head.html

I can investigate removal of feed_meta, etc. for our site